.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,19 @@
+name: "Spell check"
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  check:
+    # run on all push events or on PR syncs not from the same repo
+    if: github.event_name != 'pull_request' || github.repository != github.event.pull_request.head.repo.full_name
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Spell check
+        uses: crate-ci/typos@v1.18.2
+        with:
+          config: ./.typos.toml
```

Comment:
Could it be moved into the .GitHub directory to avoid clutter, or can it be used "by default" by running the program locally

Reply:
the CLI searches for it by default, so yes, it would be used if someone just locally ran `typos` in the directory without options